api/permissions/migrations/0010_add_manage_tags_permission.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.2.15 on 2024-09-13 16:18
+from django.apps.registry import Apps
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+from permissions.models import PROJECT_PERMISSION_TYPE
+from projects.permissions import MANAGE_TAGS
+
+
+def add_manage_tags_permission(apps: Apps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    permission_model_class = apps.get_model("permissions", "permissionmodel")
+    permission_model_class.objects.get_or_create(
+        key=MANAGE_TAGS,
+        description="Allows the user to manage tags in the given project.",
+        type=PROJECT_PERMISSION_TYPE,
+    )
+
+
+def reverse(apps: Apps, schema_editor: BaseDatabaseSchemaEditor) -> None:  # pragma: no cover
```

Comment:
Justification for adding `pragma: no cover` here:

Considering the complexity of the code, and the very limited scope in which this code will ever be executed, I figured that it wasn't worth writing a migration test for it (since they are quite arduous to write). 

I have tested the migration forwards and backwards manually.